packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -64,8 +64,6 @@ describe('e2e api calls', () => {
       expect(currentSession.eraLength.toNumber()).toBeGreaterThanOrEqual(0);
       expect(currentSession.eraProgress.toNumber()).toBeGreaterThanOrEqual(0);
       expect(currentSession.isEpoch).toBe(true);
-      expect(currentSession.lastEraLengthChange.toNumber()).toBeGreaterThanOrEqual(0);
```

Comment:
support for last era length change and last length change is removed from polkadots